ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activity-text-input/activityTextInput.component.ts

```diff
@@ -104,12 +104,17 @@ export class ActivityTextInput implements OnInit, OnChanges, OnDestroy {
   private buildForm(): void {
     this.controlName = this.block.stableId;
 
-    const formGroup = new FormGroup({
-      [this.controlName]: new FormControl({
-        value: this.block.answer,
-        disabled: this.readonly,
-      }),
-    });
+    const formGroup = new FormGroup(
+      {
+        [this.controlName]: new FormControl({
+          value: this.block.answer,
+          disabled: this.readonly,
+        }),
+      },
+      {
+        updateOn: this.block.hasUniqueValueValidator ? 'blur' : 'change'
```

Comment:
Invoke the validation on blur for questions with **UNIQUE_VALUES** validation (for optimal performance)

Reply:
Seems good, what is this used for? Does it blur the question?

Reply:
> Does it blur the question?

No. It means that the `FormControl` updates on the input **blur** event only 
(not for every symbol typing, but only when the input field loses focus).
 We should apply the `onBlur` update for some inputs that have **_UNIQUE_VALUE_** validation rule 
(in order to decrease an amount of requests to the server side - because this particular validation is quite a costly operation on BE side)